symphony/app/fbcnms-projects/magmalte/app/components/KPIGrid.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import DeviceStatusCircle from '@fbcnms/ui/components/icons/DeviceStatusCircle';
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+
+type KPIData = {
+  category: string,
+  value: number | string,
+  unit?: string,
+  statusCircle: boolean,
+  status?: 'Disabled' | 'Up' | 'Down',
+};
+
+export type KPIRows = KPIData[];
+
+type Props = {data: KPIRows[]};
+
+export default function KPIGrid(props: Props) {
```

Comment:
@HannaFar  minor nit: This component could be merged with KPI Tray component or replace KPI Tray component.